Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,295 @@
+                        model.clearBufferPoints()
+                        model.status = .addPoints
+                    }
+                }
+            }
+            .alert("Buffer Radius", isPresented: $model.inputBoxIsPresented, actions: {
+                TextField("100", text: $model.radiusInput)
+                    .keyboardType(.numberPad)
+                // Input done button.
+                Button("Done", action: {
+                    model.addBufferPoint()
+                    model.radiusInput = ""
+                    model.drawBuffers()
+                    model.status = Status.bufferCreated
+                })
+                // Input cancel button.
+                Button("Cancel", role: .cancel, action: {
+                    model.radiusInput = ""
+                })
+                // Input box message.
+            }, message: {
+                Text("Please enter a number between 0 and 300 miles.")
+            })
+    }
+}
+
+private extension CreateBuffersAroundPointsView {
+    // The view model for this sample.
+    private class Model: ObservableObject {
+        /// A Boolean value indicating whether union is on.
+        @Published var shouldUnion = false
+        
+        /// A Point representing where the user has tapped on the map.
+        @Published var tappedPoint: Point!
+        
+        /// A Boolean value indicating whether the input box is showing.
+        @Published var inputBoxIsPresented = false
+        
+        /// The input obtained from the user for the buffer radius of a point.
+        @Published var radiusInput: String = ""
+        
+        /// The status of the sample.
+        @Published var status = Status.addPoints
+        
+        /// A Map centered on Texas with image layers.
+        var map = Map()
+        
+        /// An Array for the graphic overlays.
+        var graphicsOverlays: [GraphicsOverlay] = []
+        
+        /// An Array for the tapped points and their radii.
+        var bufferPoints: [(point: Point, radius: Double)] = []
+        
+        /// A GraphicsOverlay for the boundary around the valid area of use.
+        private var boundaryGraphicsOverlay: GraphicsOverlay
+        
+        /// A GraphicsOverlay for the points' buffers.
+        private var bufferGraphicsOverlay: GraphicsOverlay
+        
+        /// A GraphicsOverlay for the points of the tapped locations.
+        private var tappedPointsGraphicsOverlay: GraphicsOverlay
+        
+        /// A Polygon that represents the valid area of use for the spatial reference.
+        private let boundaryPolygon: Polygon
+        
+        init() {
+            /// The spatial reference for this sample.
+            let statePlaneNorthCentralTexas = SpatialReference(wkid: WKID(32038)!)!
+            
+            // Create boundary polygon.
+            boundaryPolygon = {
+                let boundaryPoints = [
+                    Point(x: -103.070, y: 31.720, spatialReference: .wgs84),
```

Comment:
```swift
                    Point(latitude: 31.720, longitude: -103.070),
```
We use this convenience initializer when it comes to WGS84, so it is more obvious to non-GIS users.